ayon_api/server_api.py

```diff
@@ -1580,7 +1622,7 @@ def upload_file_from_stream(
             requests.Response: Response object
 
         """
-        url = self._endpoint_to_url(endpoint)
+        url = self._endpoint_to_url(endpoint, use_rest=False)
```

Comment:
This was changed back to original value in `develop` after merge [commit](https://github.com/ynput/ayon-python-api/commit/e9b7fe12f40e4eed539d11998b7e05f8f0389692)